table.cpp

```diff
@@ -124,11 +128,11 @@ TableCell::TableCell(float d, const std::string& c, int precision, const std::st
   } else {
    inquiry = bad_status = override = extension = version_conflict = cancelled = false;
   }
-    
 }
 
 
 
+
```

Comment:
Unnecessary addition?

Reply:
Fixed in [07ac6de](https://github.com/Submitty/RainbowGrades/pull/77/commits/07ac6dea469083e1954b9544df5eb49deea68582)